scripts/devel/create_devel_task_generator.py

```diff
@@ -99,45 +99,52 @@ def main(argv=sys.argv[1:]):
 
```

Comment:
Nothing to critique. Just want to call attention to another change I think could have merited its own PR to cut down on the review burden and feature / noise ratio. Breaking this out would have also gotten it more testing on devel and PR jobs that aren't CI.